cime_config/config_pes.xml

```diff
@@ -478,203 +620,166 @@
         </nthrds>
         <rootpe>
           <rootpe_atm>0</rootpe_atm>
-          <rootpe_lnd>0</rootpe_lnd>
-          <rootpe_rof>16</rootpe_rof>
-          <rootpe_ice>32</rootpe_ice>
-          <rootpe_ocn>128</rootpe_ocn>
-          <rootpe_glc>0</rootpe_glc>
-          <rootpe_wav>0</rootpe_wav>
+          <rootpe_lnd>1</rootpe_lnd>
+          <rootpe_rof>1</rootpe_rof>
+          <rootpe_ice>2</rootpe_ice>
+          <rootpe_ocn>32</rootpe_ocn>
+          <rootpe_glc>1</rootpe_glc>
+          <rootpe_wav>1</rootpe_wav>
           <rootpe_cpl>0</rootpe_cpl>
         </rootpe>
       </pes>
     </mach>
   </grid>
 
-
-  <!-- Configurations for the two degree tripolar (tnx2) grid for compsets including iHAMOCC  !-->
-  
-  <grid name="a%T62.+oi%tnx2v1">
-    <mach name="any">
-      <pes pesize="M" compset="_DATM.*_BLOM">
-	<comment>Medium sized pe-layout with  pes in total</comment>
-	<ntasks>
-	  <ntasks_atm>15</ntasks_atm> 
-	  <ntasks_rof>10</ntasks_rof> 
-	  <ntasks_ice>76</ntasks_ice> 
-	  <ntasks_ocn>155</ntasks_ocn> 
-	  <ntasks_cpl>76</ntasks_cpl> 
-	  <ntasks_lnd>1</ntasks_lnd>
-	  <ntasks_glc>1</ntasks_glc> 
-	  <ntasks_wav>1</ntasks_wav> 
-	</ntasks>
-	<nthrds>
-	  <nthrds_atm>1</nthrds_atm>                   
-	  <nthrds_lnd>1</nthrds_lnd> 
-	  <nthrds_rof>1</nthrds_rof> 
-	  <nthrds_ice>1</nthrds_ice> 
-	  <nthrds_ocn>1</nthrds_ocn> 
-	  <nthrds_glc>1</nthrds_glc> 
-	  <nthrds_wav>1</nthrds_wav> 
-	  <nthrds_cpl>1</nthrds_cpl> 
-	</nthrds>
-	<rootpe>
-	  <rootpe_atm>76</rootpe_atm> 
-	  <rootpe_lnd>0</rootpe_lnd> 
-	  <rootpe_rof>91</rootpe_rof> 
-	  <rootpe_ice>0</rootpe_ice>    
-	  <rootpe_ocn>101</rootpe_ocn>   
-	  <rootpe_glc>0</rootpe_glc> 
-	  <rootpe_wav>0</rootpe_wav> 
-	  <rootpe_cpl>0</rootpe_cpl>                         
-	</rootpe>
-      </pes>
-    </mach>
-  </grid>
   <grid name="a%T62.+oi%tnx2v1">
     <mach name="any">
-      <pes pesize="XS" compset="_DATM.*_BLOM%ECO">
-	<comment>Very small and efficent pe-layout on 128 pes in total</comment>
-	<ntasks>
-	  <ntasks_atm>1</ntasks_atm> 
-	  <ntasks_rof>1</ntasks_rof> 
-	  <ntasks_ice>30</ntasks_ice> 
-	  <ntasks_ocn>96</ntasks_ocn> 
-	  <ntasks_cpl>32</ntasks_cpl> 
-	  <ntasks_lnd>1</ntasks_lnd>
-	  <ntasks_glc>1</ntasks_glc> 
-	  <ntasks_wav>1</ntasks_wav> 
-	</ntasks>
-	<nthrds>
-	  <nthrds_atm>1</nthrds_atm>                   
-	  <nthrds_lnd>1</nthrds_lnd> 
-	  <nthrds_rof>1</nthrds_rof> 
-	  <nthrds_ice>1</nthrds_ice> 
-	  <nthrds_ocn>1</nthrds_ocn> 
-	  <nthrds_glc>1</nthrds_glc> 
-	  <nthrds_wav>1</nthrds_wav> 
-	  <nthrds_cpl>1</nthrds_cpl> 
-	</nthrds>
-	<rootpe>
-	  <rootpe_atm>0</rootpe_atm> 
-	  <rootpe_lnd>1</rootpe_lnd> 
-	  <rootpe_rof>1</rootpe_rof> 
-	  <rootpe_ice>2</rootpe_ice>    
-	  <rootpe_ocn>32</rootpe_ocn>   
-	  <rootpe_glc>1</rootpe_glc> 
-	  <rootpe_wav>1</rootpe_wav> 
-	  <rootpe_cpl>0</rootpe_cpl>                         
-	</rootpe>
+      <pes pesize="S" compset="_DATM.*_BLOM%ECO">
```

Comment:
This is confusing: There is on pesize="S" layout with 155 ocean tasks, one pesize="M" layout with 155 ocean tasks further below, and one pesize="M" layout with 272 ocean tasks. It is a bit difficult to see what is new and what has been there before.

Reply:
These are old pe-layouts, I only added new `XS` layouts.
I agree that it is confusing, but I didn't want to remove something that someone might still be using.